src/foam/nanos/theme/Themes.js

```diff
@@ -174,9 +175,15 @@ Later themes:
         }
       }
 
+      if ( theme == null ) {
```

Comment:
Why did you add this back?

Reply:
It allows loading group theme when no spid theme.